src/allmydata/scripts/tahoe_backup.py

```diff
@@ -56,20 +56,20 @@ def put_child(dirurl, childname, childcap):
     if resp.status not in (200, 201):
         raise HTTPError("Error during put_child", resp)
 
-class BackupProcessingError(Exception):
-    pass
-
 class BackerUpper:
+    """
```

Comment:
I know lots of other classes don't do this either, but should we inherit from `object` for maximum compatibility?

Reply:
Oh yea, totally.  And since there is no Python API compatibility guarantee I can just fix these left and right.  :)